ddtrace/contrib/internal/aiopg/patch.py

```diff
@@ -0,0 +1,70 @@
+# 3p
+import aiopg.connection
+import psycopg2.extensions
+
+from ddtrace import config
+from ddtrace.contrib.aiopg.connection import AIOTracedConnection
+from ddtrace.contrib.psycopg.connection import patch_conn as psycopg_patch_conn
+from ddtrace.contrib.psycopg.extensions import _patch_extensions
+from ddtrace.contrib.psycopg.extensions import _unpatch_extensions
+from ddtrace.internal.utils.wrappers import unwrap as _u
+from ddtrace.vendor import wrapt
+
+from ...internal.schema import schematize_service_name
```

Comment:
This should import from `ddtrace.internal` and not `ddtrace.contrib.internal`.

```suggestion
from ....internal.schema import schematize_service_name
```